src/neo-vm/Debugger.cs

```diff
@@ -73,13 +75,15 @@ public VMState StepOver()
         {
             if (engine.State.HasFlag(VMState.HALT) || engine.State.HasFlag(VMState.FAULT))
                 return engine.State;
-            engine.State &= ~VMState.BREAK;
+
+            engine.State = VMState.NONE;
             int c = engine.InvocationStack.Count;
             do
             {
                 ExecuteAndCheckBreakPoints();
             }
             while (!engine.State.HasFlag(VMState.HALT) && !engine.State.HasFlag(VMState.FAULT) && !engine.State.HasFlag(VMState.BREAK) && engine.InvocationStack.Count > c);
```

Comment:
Check only none?

Reply:
No. Each step has the potential to change the state of the VM.

Reply:
but this is the same as equal none

Reply:
So?

Reply:
3 checks vs one, is better, state is not a flag now

Reply:
`engine.State != VMState.None`?

Reply:
yes